src/site/stages/build/plugins/check-broken-links/helpers/applyIgnoredRoutes.js

```diff
@@ -0,0 +1,48 @@
+function _getReactLandingPages(files) {
+  const reactLandingPages = Object.keys(files)
+    .map(fileName => files[fileName])
+    .filter(file => !!file.entryname)
+    .map(file => `/${file.path}`);
+
+  return reactLandingPages;
+}
+
+function applyIgnoredRoutes(
+  brokenPages,
+  files,
+  getReactLandingPages = _getReactLandingPages,
+) {
+  // We ignore React landing pages, because React apps have
+  // dynamic, client-side routing and only the root page
+  // will have an HTML page.
+
+  // We also ignore pages ending in .asp, because although those
+  // pages are under the same domain, they are not part of our
+  // source code, so we can't validate them.
+
+  const reactLandingPages = getReactLandingPages(files);
+
+  const filteredBrokenPages = brokenPages
+    .map(brokenPage => {
+      const filteredLinkErrors = brokenPage.linkErrors
+        .filter(linkError => !linkError.target.endsWith('.asp'))
+        .filter(
+          linkError =>
+            !reactLandingPages.some(reactPath =>
+              linkError.target.startsWith(reactPath),
+            ),
+        );
+
+      return {
+        ...brokenPage,
+        linkErrors: filteredLinkErrors,
+      };
+    })
+
+    .filter(brokenPage => brokenPage.linkErrors.length > 0);
+
+  return filteredBrokenPages;
+}
+
+module.exports = applyIgnoredRoutes;
+module.exports.getReactLandingPages = _getReactLandingPages;
```

Comment:
After finding all of the link errors, we then filter some out -

1. Child routes of React apps, because they may be routed to via JS routing
2. Pages ending with `.asp`, because there are many of those under VA.gov, but they aren't in our local source code.

Without this, we get these errors -

```

---

There are 135 broken links!

---

There are 1 broken links on health-care/about-va-health-benefits/your-care-team:

<a href="/HEALTHBENEFITS/cost/index.asp" id="ghefhsbq">Your health care costs</a>

---

There are 1 broken links on health-care/about-va-health-benefits/where-you-go-for-care:

<a href="/HEALTHBENEFITS/cost/index.asp" id="ggjrhd">Your health care costs</a>

---

There are 1 broken links on health-care/about-va-health-benefits/dental-care:

<a href="/HEALTHBENEFITS/cost/index.asp" id="bohzwplwh">Your health care costs</a>

---

There are 1 broken links on health-care/about-va-health-benefits/vision-care:

<a href="/HEALTHBENEFITS/cost/index.asp" id="lbmgmtwk">Your health care costs</a>

---

There are 7 broken links on health-care/after-you-apply:

<a href="/HEALTHBENEFITS/cost/index.asp" id="fnaboqhom">Your health care costs</a>
<a href="/HEALTHBENEFITS/apply/active_duty.asp" id="zezfogt">Active-duty service members</a>
<a href="/HEALTHBENEFITS/apply/returning_servicemembers.asp" id="mvlzkpeqp">Returning service members</a>
<a href="/HEALTHBENEFITS/vtp/Beneficiary_Travel.asp" id="mrengsdfe">Get travel pay</a>
<a href="/HEALTHBENEFITS/vhic/index.asp" id="nocwlsijpf">Request a health ID card</a>
<a href="/health/topics/index.asp" id="qwznuteeo">Health topics A-Z</a>
<a href="/health/rights/patientrights.asp" id="jkpextfg">Patient rights and responsibilities</a>

---

There are 7 broken links on health-care/refill-track-prescriptions:

<a href="/HEALTHBENEFITS/cost/index.asp" id="uubkyglg">Your health care costs</a>
<a href="/HEALTHBENEFITS/apply/active_duty.asp" id="zcganqtjsr">Active-duty service members</a>
<a href="/HEALTHBENEFITS/apply/returning_servicemembers.asp" id="tvrbyiguda">Returning service members</a>
<a href="/HEALTHBENEFITS/vtp/Beneficiary_Travel.asp" id="hdqiqnwicl">Get travel pay</a>
<a href="/HEALTHBENEFITS/vhic/index.asp" id="ggibgcnn">Request a health ID card</a>
<a href="/health/topics/index.asp" id="ablmdurhpk">Health topics A-Z</a>
<a href="/health/rights/patientrights.asp" id="rzxwllmue">Patient rights and responsibilities</a>

---

There are 7 broken links on health-care/secure-messaging:

<a href="/HEALTHBENEFITS/cost/index.asp" id="wznmdbqh">Your health care costs</a>
<a href="/HEALTHBENEFITS/apply/active_duty.asp" id="kscuzrfvil">Active-duty service members</a>
<a href="/HEALTHBENEFITS/apply/returning_servicemembers.asp" id="rnhfa">Returning service members</a>
<a href="/HEALTHBENEFITS/vtp/Beneficiary_Travel.asp" id="tutpcdm">Get travel pay</a>
<a href="/HEALTHBENEFITS/vhic/index.asp" id="icujrqzsoc">Request a health ID card</a>
<a href="/health/topics/index.asp" id="ilpuzjf">Health topics A-Z</a>
<a href="/health/rights/patientrights.asp" id="goprgewta">Patient rights and responsibilities</a>

---

There are 7 broken links on health-care/schedule-view-va-appointments:

<a href="/HEALTHBENEFITS/cost/index.asp" id="wobczcfan">Your health care costs</a>
<a href="/HEALTHBENEFITS/apply/active_duty.asp" id="onaxwbo">Active-duty service members</a>
<a href="/HEALTHBENEFITS/apply/returning_servicemembers.asp" id="vryahxvj">Returning service members</a>
<a href="/HEALTHBENEFITS/vtp/Beneficiary_Travel.asp" id="wqamkz">Get travel pay</a>
<a href="/HEALTHBENEFITS/vhic/index.asp" id="kjnzsliiv">Request a health ID card</a>
<a href="/health/topics/index.asp" id="hdhiieve">Health topics A-Z</a>
<a href="/health/rights/patientrights.asp" id="nigrcvay">Patient rights and responsibilities</a>

---

There are 7 broken links on health-care/view-test-and-lab-results:

<a href="/HEALTHBENEFITS/cost/index.asp" id="wlgyotf">Your health care costs</a>
<a href="/HEALTHBENEFITS/apply/active_duty.asp" id="gugmdqhijz">Active-duty service members</a>
<a href="/HEALTHBENEFITS/apply/returning_servicemembers.asp" id="zuohtfidx">Returning service members</a>
<a href="/HEALTHBENEFITS/vtp/Beneficiary_Travel.asp" id="stuysedvj">Get travel pay</a>
<a href="/HEALTHBENEFITS/vhic/index.asp" id="cicdtyvukr">Request a health ID card</a>
<a href="/health/topics/index.asp" id="uhkfmskglb">Health topics A-Z</a>
<a href="/health/rights/patientrights.asp" id="ujmunad">Patient rights and responsibilities</a>

---

There are 7 broken links on health-care/health-needs-conditions:

<a href="/HEALTHBENEFITS/cost/index.asp" id="unnuzoh">Your health care costs</a>
<a href="/HEALTHBENEFITS/apply/active_duty.asp" id="xbmilkf">Active-duty service members</a>
<a href="/HEALTHBENEFITS/apply/returning_servicemembers.asp" id="xlywnwalp">Returning service members</a>
<a href="/HEALTHBENEFITS/vtp/Beneficiary_Travel.asp" id="lhjwppkab">Get travel pay</a>
<a href="/HEALTHBENEFITS/vhic/index.asp" id="kveutagc">Request a health ID card</a>
<a href="/health/topics/index.asp" id="gtqfkxmqt">Health topics A-Z</a>
<a href="/health/rights/patientrights.asp" id="nwmitc">Patient rights and responsibilities</a>

---

There are 1 broken links on health-care/health-needs-conditions/health-issues-related-to-service-era:

<a href="/health/topics/index.asp" id="saikvaqknb">Health topics A-Z</a>

---

There are 1 broken links on health-care/health-needs-conditions/mental-health:

<a href="/health/topics/index.asp" id="wjrkplr">Health topics A-Z</a>

---

There are 1 broken links on health-care/health-needs-conditions/military-sexual-trauma:

<a href="/health/topics/index.asp" id="mzhijjyer">Health topics A-Z</a>

---

There are 1 broken links on health-care/health-needs-conditions/substance-use-problems:

<a href="/health/topics/index.asp" id="fahf">Health topics A-Z</a>

---

There are 1 broken links on health-care/health-needs-conditions/chemical-hazardous-materials-exposure:

<a href="/health/topics/index.asp" id="nzudyuf">Health topics A-Z</a>

---

There are 1 broken links on health-care/health-needs-conditions/womens-health-needs:

<a href="/health/topics/index.asp" id="bwexbsyoan">Health topics A-Z</a>

---

There are 7 broken links on health-care/wellness-programs:

<a href="/HEALTHBENEFITS/cost/index.asp" id="hxxaicg">Your health care costs</a>
<a href="/HEALTHBENEFITS/apply/active_duty.asp" id="kvkivped">Active-duty service members</a>
<a href="/HEALTHBENEFITS/apply/returning_servicemembers.asp" id="btmdtrend">Returning service members</a>
<a href="/HEALTHBENEFITS/vtp/Beneficiary_Travel.asp" id="rprttns">Get travel pay</a>
<a href="/HEALTHBENEFITS/vhic/index.asp" id="htxpvsti">Request a health ID card</a>
<a href="/health/topics/index.asp" id="ysgftzc">Health topics A-Z</a>
<a href="/health/rights/patientrights.asp" id="gzbvcuai">Patient rights and responsibilities</a>

---

There are 1 broken links on health-care/about-va-health-benefits/long-term-care:

<a href="/HEALTHBENEFITS/cost/index.asp" id="cpkowdxb">Your health care costs</a>

---

There are 7 broken links on health-care/update-health-information:

<a href="/HEALTHBENEFITS/cost/index.asp" id="ouqfekcic">Your health care costs</a>
<a href="/HEALTHBENEFITS/apply/active_duty.asp" id="snqhsyp">Active-duty service members</a>
<a href="/HEALTHBENEFITS/apply/returning_servicemembers.asp" id="cvhdtvab">Returning service members</a>
<a href="/HEALTHBENEFITS/vtp/Beneficiary_Travel.asp" id="iafiplx">Get travel pay</a>
<a href="/HEALTHBENEFITS/vhic/index.asp" id="tpymntspr">Request a health ID card</a>
<a href="/health/topics/index.asp" id="xwnivco">Health topics A-Z</a>
<a href="/health/rights/patientrights.asp" id="krjnnufalv">Patient rights and responsibilities</a>

---

There are 7 broken links on health-care/about-affordable-care-act:

<a href="/HEALTHBENEFITS/cost/index.asp" id="oxytudsqc">Your health care costs</a>
<a href="/HEALTHBENEFITS/apply/active_duty.asp" id="xxxqptum">Active-duty service members</a>
<a href="/HEALTHBENEFITS/apply/returning_servicemembers.asp" id="dswzzweu">Returning service members</a>
<a href="/HEALTHBENEFITS/vtp/Beneficiary_Travel.asp" id="venvytwo">Get travel pay</a>
<a href="/HEALTHBENEFITS/vhic/index.asp" id="osylgzqll">Request a health ID card</a>
<a href="/health/topics/index.asp" id="vsnwhqd">Health topics A-Z</a>
<a href="/health/rights/patientrights.asp" id="owddyscgni">Patient rights and responsibilities</a>

---

There are 7 broken links on health-care/family-caregiver-benefits:

<a href="/HEALTHBENEFITS/cost/index.asp" id="zmdghhjvtd">Your health care costs</a>
<a href="/HEALTHBENEFITS/apply/active_duty.asp" id="jmcicsxhbc">Active-duty service members</a>
<a href="/HEALTHBENEFITS/apply/returning_servicemembers.asp" id="mteusjo">Returning service members</a>
<a href="/HEALTHBENEFITS/vtp/Beneficiary_Travel.asp" id="mzitolbec">Get travel pay</a>
<a href="/HEALTHBENEFITS/vhic/index.asp" id="cakmnt">Request a health ID card</a>
<a href="/health/topics/index.asp" id="pzvfejue">Health topics A-Z</a>
<a href="/health/rights/patientrights.asp" id="eidonkzxus">Patient rights and responsibilities</a>

---

There are 7 broken links on health-care/order-hearing-aid-batteries-prosthetic-socks:

<a href="/HEALTHBENEFITS/cost/index.asp" id="fvsfhhmywc">Your health care costs</a>
<a href="/HEALTHBENEFITS/apply/active_duty.asp" id="ewrtscmzcl">Active-duty service members</a>
<a href="/HEALTHBENEFITS/apply/returning_servicemembers.asp" id="bybepiizv">Returning service members</a>
<a href="/HEALTHBENEFITS/vtp/Beneficiary_Travel.asp" id="lywaai">Get travel pay</a>
<a href="/HEALTHBENEFITS/vhic/index.asp" id="qlthkcte">Request a health ID card</a>
<a href="/health/topics/index.asp" id="afyzztxln">Health topics A-Z</a>
<a href="/health/rights/patientrights.asp" id="knniazvcy">Patient rights and responsibilities</a>

---

There are 7 broken links on health-care/how-to-apply:

<a href="/HEALTHBENEFITS/cost/index.asp" id="lizbrweo">Your health care costs</a>
<a href="/HEALTHBENEFITS/apply/active_duty.asp" id="neftnfzgg">Active-duty service members</a>
<a href="/HEALTHBENEFITS/apply/returning_servicemembers.asp" id="cnwtndy">Returning service members</a>
<a href="/HEALTHBENEFITS/vtp/Beneficiary_Travel.asp" id="vsipqmshed">Get travel pay</a>
<a href="/HEALTHBENEFITS/vhic/index.asp" id="gzrkkhq">Request a health ID card</a>
<a href="/health/topics/index.asp" id="vynxbsgupk">Health topics A-Z</a>
<a href="/health/rights/patientrights.asp" id="cekroix">Patient rights and responsibilities</a>

---

There are 7 broken links on get-a-veteran-health-identification-card-vhic:

<a href="/HEALTHBENEFITS/cost/index.asp" id="oagxcno">Your health care costs</a>
<a href="/HEALTHBENEFITS/apply/active_duty.asp" id="yzqtbcl">Active-duty service members</a>
<a href="/HEALTHBENEFITS/apply/returning_servicemembers.asp" id="dumsawrsk">Returning service members</a>
<a href="/HEALTHBENEFITS/vtp/Beneficiary_Travel.asp" id="ekwnnond">Get travel pay</a>
<a href="/HEALTHBENEFITS/vhic/index.asp" id="newcqh">Request a health ID card</a>
<a href="/health/topics/index.asp" id="vadgj">Health topics A-Z</a>
<a href="/health/rights/patientrights.asp" id="zjursnewa">Patient rights and responsibilities</a>

---

There are 7 broken links on health-care/get-medical-records:

<a href="/HEALTHBENEFITS/cost/index.asp" id="qtvljwk">Your health care costs</a>
<a href="/HEALTHBENEFITS/apply/active_duty.asp" id="talzybrz">Active-duty service members</a>
<a href="/HEALTHBENEFITS/apply/returning_servicemembers.asp" id="khlguj">Returning service members</a>
<a href="/HEALTHBENEFITS/vtp/Beneficiary_Travel.asp" id="gjnnftkz">Get travel pay</a>
<a href="/HEALTHBENEFITS/vhic/index.asp" id="upvvtzv">Request a health ID card</a>
<a href="/health/topics/index.asp" id="xudmczf">Health topics A-Z</a>
<a href="/health/rights/patientrights.asp" id="gsltlvbha">Patient rights and responsibilities</a>

---

There are 1 broken links on pension:

<a href="/pension/application/527EZ/introduction" id="pvhdpoht">
            <span class="hub-page-link-list__header">Apply for a Veterans Pension online (VA Form 21P-527EZ)</span>
                <img class="all-link-arrow" src="/img/arrow-right-blue.svg" alt="right-arrow">
        
            <p class="va-nav-linkslist-description">Complete your application online now.</p>
    </a>

---

There are 3 broken links on health-care/eligibility/priority-groups:

<a href="/HEALTHBENEFITS/apply/active_duty.asp" id="wxjhzhpdv">Active-duty service members</a>
<a href="/HEALTHBENEFITS/apply/returning_servicemembers.asp" id="uwufktfbjy">Returning service members</a>
<a href="/disability-benefits/apply/form-526-disability-claim/introduction">File for increased disability compensation</a>

---

There are 1 broken links on health-care/about-va-health-benefits/va-health-care-and-other-insurance:

<a href="/HEALTHBENEFITS/cost/index.asp" id="emrczcev">Your health care costs</a>

---

There are 7 broken links on health-care/eligibility:

<a href="/HEALTHBENEFITS/cost/index.asp" id="zksoagv">Your health care costs</a>
<a href="/HEALTHBENEFITS/apply/active_duty.asp" id="matpdbga">Active-duty service members</a>
<a href="/HEALTHBENEFITS/apply/returning_servicemembers.asp" id="nymgwkul">Returning service members</a>
<a href="/HEALTHBENEFITS/vtp/Beneficiary_Travel.asp" id="evluquwxf">Get travel pay</a>
<a href="/HEALTHBENEFITS/vhic/index.asp" id="yjhsvl">Request a health ID card</a>
<a href="/health/topics/index.asp" id="mamgrqiww">Health topics A-Z</a>
<a href="/health/rights/patientrights.asp" id="hccgaih">Patient rights and responsibilities</a>

---

There are 7 broken links on health-care/about-va-health-benefits:

<a href="/HEALTHBENEFITS/cost/index.asp" id="nsrnvbtja">Your health care costs</a>
<a href="/HEALTHBENEFITS/apply/active_duty.asp" id="gvimgirw">Active-duty service members</a>
<a href="/HEALTHBENEFITS/apply/returning_servicemembers.asp" id="ohfarybc">Returning service members</a>
<a href="/HEALTHBENEFITS/vtp/Beneficiary_Travel.asp" id="bzxqvrqqr">Get travel pay</a>
<a href="/HEALTHBENEFITS/vhic/index.asp" id="uytcvwlnh">Request a health ID card</a>
<a href="/health/topics/index.asp" id="awkk">Health topics A-Z</a>
<a href="/health/rights/patientrights.asp" id="vmkakvbi">Patient rights and responsibilities</a>

---

There are 1 broken links on health-care:

<a href="/health-care/apply/application/introduction" id="jtitqc">
            <span class="hub-page-link-list__header">Apply online (VA Form 10-10EZ)</span>
                <img class="all-link-arrow" src="/img/arrow-right-blue.svg" alt="right-arrow">
        
            <p class="va-nav-linkslist-description">Start your Veterans health care application now.</p>
    </a>

---

There are 2 broken links on education/change-gi-bill-benefits:

<a href="/education/apply-for-education-benefits/application/1995/introduction">Complete VA Form 22-1995 online</a>
<a href="/education/apply-for-education-benefits/application/5495/introduction">Complete VA Form 22-5495 online</a>

---

There are 1 broken links on education/transfer-post-9-11-gi-bill-benefits:

<a href="/education/apply-for-education-benefits/application/1990E/introduction">Apply online now</a>

---

There are 1 broken links on education/other-va-education-benefits/veap:

<a href="/education/apply-for-education-benefits/application/1990/introduction/">Apply for education benefits online now</a>

---

There are 1 broken links on education/survivor-dependent-benefits/fry-scholarship:

<a href="/education/apply-for-education-benefits/application/5490/introduction">Apply online now</a>

---

There are 1 broken links on education/survivor-dependent-benefits/dependents-education-assistance:

<a href="/education/apply-for-education-benefits/application/5490/introduction">Apply online now</a>

---
```